WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -1546,43 +1546,31 @@ local function modify(parent, region, data)
         region:RealClearAllPoints()
         region:SetPoint("CENTER", UIParent, "CENTER")
       end
-      for active, regionData in ipairs(self.sortedChildren) do
+      for _, regionData in ipairs(self.sortedChildren) do
         if regionData.shown then
-          numVisible = numVisible + 1
-          local childRegion = regionData.region
-          local regionLeft, regionRight, regionTop, regionBottom
-             = SafeGetPos(childRegion, childRegion.GetLeft), SafeGetPos(childRegion, childRegion.GetRight),
-               SafeGetPos(childRegion, childRegion.GetTop), SafeGetPos(childRegion, childRegion.GetBottom)
-
-          if(regionLeft and regionRight and regionTop and regionBottom) then
-            minX = minX and min(regionLeft, minX) or regionLeft
-            maxX = maxX and max(regionRight, maxX) or regionRight
-            minY = minY and min(regionBottom, minY) or regionBottom
-            maxY = maxY and max(regionTop, maxY) or regionTop
-          end
+          anyVisible = true
+          break
         end
       end
 
-      if numVisible > 0 then
+      if anyVisible then
         self:Show()
-        minX, maxX, minY, maxY = (minX or 0), (maxX or 0), (minY or 0), (maxY or 0)
-
-        local width, height = maxX - minX, maxY - minY
+        self:SetSize(1, 1)
+        local left, bottom, width, height = self:GetBoundsRect()
         width = width > 0 and width or 16
         height = height > 0 and height or 16
 
-        self:SetWidth(width)
-        self:SetHeight(height)
+        self:SetSize(width, height)
         self.currentWidth = width
         self.currentHeight = height
 
         if data.border and not data.useAnchorPerUnit then
-          local regionLeft = SafeGetPos(region, region.GetLeft) or minX
-          local regionBottom = SafeGetPos(region, region.GetBottom) or minY
+          local regionLeft = SafeGetPos(region, region.GetLeft) or left
+          local regionBottom = SafeGetPos(region, region.GetBottom) or bottom
```

Comment:
Do you think we still need to use region.GetLeft/GetBottom instead of left & bottom?